packages/ui-extensions/src/surfaces/checkout/style/examples/hiding.example.tsx

```diff
@@ -0,0 +1,8 @@
+<View
```

Comment:
is there a reason for not providing an example in JS?

Also should we provide the example imports for the example to make it even easier for developers to just copy and paste to get a good starting point ? 

```tsx
import {
  	reactExtension,
  	Style,
  	View,
} from '@shopify/ui-extensions-react/checkout';


export default reactExtension('purchase.checkout.block.render', () => (
  	<Extension />
));

...

```